src/style.css

```diff
@@ -1,3 +1,71 @@
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: 100vw;
+  background-color: white;
+}
+
 body {
   margin: 0;
 }
+
+.img_link {
+  margin-left: 50px;
```

Comment:
better to set on header(to make it like box that contain content inside)
padding: 0 50px;